lib/FluxibleContext.js

```diff
@@ -83,12 +84,17 @@ FluxContext.prototype.plug = function (plugin) {
  * @param {Object} context FluxContext object
  * @param {Function} action Action to call
  * @param {Object} payload Payload for the action
+ * @param {Object} meta Metadata to keep track of the current flow of executing actions
  * @param {Function} resolve function to call on success
  * @param {Function} reject function to call on failure
  * @private
  */
-function executeActionInternal(context, action, payload, resolve, reject) {
-    var syncResult = action(context.getActionContext(), payload, function (err, result) {
+function executeActionInternal(context, action, payload, meta, resolve, reject) {
+    var actionContext = context.getActionContext(meta);
+    var displayName = action.displayName || action.name;
+    debug('Executing action ' + displayName + ' with trace', actionContext.stack.join('.'));
```

Comment:
minor: only thing i dont like is that this join happens even if debug isn't used